src/quantum/README.rst

```diff
@@ -0,0 +1,188 @@
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+   The console will output the information about the job, including the ID of the job.
+
+
+7. You can use the ID of the job to track its status.
+
+   .. code-block::
+
+      az quantum job show -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+
+      Id                                    State    Target          Submission time
+      ------------------------------------  -------  --------------  ---------------------------------
+      yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+
+
+8. Once the job finishes you can visualize the job's results with `az quantum job output`:
+
+   .. code-block::
+
+      az quantum job output -id yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy -o table
+   
+      Result    Frequency
+      --------  -----------  -------------------------
+      [0,0]     0.25000000   ▐█████                  |
+      [1,0]     0.25000000   ▐█████                  |
+      [0,1]     0.25000000   ▐█████                  |
+      [1,1]     0.25000000   ▐█████                  |
+
+
+Submitting Q# programs for execution from the command line
+==========================================================
+
+Prerequisites
+-------------
+
+- You need to have an Azure Quantum workspace in your subscription.
+- Install the [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/standalone), if you haven't already.
+
+
+Write your quantum application
+------------------------------
+
+First you need to have the Q# quantum application that you want to execute in
+Azure Quantum.
+
+.. tip::
+   If this is the first time for you to create Q# quantum applications, you can learn how
+   in our [Microsoft Learn module](https://docs.microsoft.com/en-us/learn/modules/qsharp-create-first-quantum-development-kit/).
+
+In this case we will use a simple quantum random bit generator. We create a Q#
+project and substitute the content of `Program.qs` with the following code:
+
+.. code-block::
+
+   namespace RandomBit {
+
+       open Microsoft.Quantum.Canon;
+       open Microsoft.Quantum.Intrinsic;
+       open Microsoft.Quantum.Measurement;
+
+       @EntryPoint()
+       operation GenerateRandomBit() : Result {
+           using (q = Qubit()) {
+               H(q);
+               return MResetZ(q);
+           }
+       }
+   }
+
+Note that the `@EntryPoint` attribute tells Q# which operation to run when the program starts.
+
+Submit the job
+--------------
+
+In this example we are going to use IonQ as the provider and the
+`ionq.simulator` as target. To submit the job to the currently selected
+default quantum workspace `az quantum job submit`:
+
+.. code-block::
+
+   az quantum job submit --target-id ionq.simulator --job-name Hello -o table
```

Comment:
ValidationError: Missing resource-group argument
ValidationError: Missing workspace-name argument
ValidationError: Missing location argument

Reply:
How can I check available `--target-id` that I can use?

Reply:
Help message `--target-id -t      : Target id.` does not provide help of target id.

Reply:
Unhandled exception make users think it is a bug.
```
Unhandled exception: Microsoft.Azure.Quantum.Client.Models.RestErrorException: Operation returned an invalid status code 'Unauthorized'
   at Microsoft.Azure.Quantum.Client.StorageOperations.SasUriWithHttpMessagesAsync(BlobDetails blobDetails, Dictionary`2 customHeaders, CancellationToken cancellationToken)
   at Microsoft.Azure.Quantum.Client.StorageOperationsExtensions.SasUriAsync(IStorageOperations operations, BlobDetails blobDetails, CancellationToken cancellationToken)
   at Microsoft.Azure.Quantum.Workspace.GetSasUriAsync(String containerName, String blobName, CancellationToken cancellationToken)
   at Microsoft.Azure.Quantum.Storage.LinkedStorageJobHelper.GetContainerClient(String containerName, CancellationToken cancellationToken)
   at Microsoft.Azure.Quantum.Storage.LinkedStorageJobHelper.UploadJobInputAsync(String jobId, Stream input, CancellationToken cancellationToken)
   at Microsoft.Quantum.Providers.Core.AzureQuantumMachine`1.SubmitAsync[QInput,QOutput](EntryPointInfo`2 info, QInput input, IQuantumMachineSubmissionContext submissionContext, ConfigureJob configureJobCallback)
   at Microsoft.Quantum.Providers.Core.AzureQuantumMachine`1.SubmitAsync[QInput,QOutput](EntryPointInfo`2 info, QInput input, IQuantumMachineSubmissionContext submissionContext)
   at Microsoft.Quantum.EntryPointDriver.Azure.SubmitJob[TIn,TOut](IQuantumMachine machine, IEntryPoint`2 entryPoint, TIn input, AzureSettings settings)
   at Microsoft.Quantum.EntryPointDriver.Azure.Submit[TIn,TOut](IEntryPoint`2 entryPoint, ParseResult parseResult, AzureSettings settings)
   at Microsoft.Quantum.EntryPointDriver.Driver`3.Submit(ParseResult parseResult, AzureSettings azureSettings)
   at System.CommandLine.Invocation.CommandHandler.GetResultCodeAsync(Object value, InvocationContext context)
   at System.CommandLine.Invocation.ModelBindingCommandHandler.InvokeAsync(InvocationContext context)
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseParseErrorReporting>b__20_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass15_0.<<UseHelp>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass24_0.<<UseVersionOption>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__21_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseParseDirective>b__19_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseDebugDirective>b__11_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__10_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass13_0.<<UseExceptionHandler>b__0>d.MoveNext()
Failed to submit job.
```

Reply:
With regards to the argument, the same comment above regarding step 4 applies. 

With respect to the target id, I expanded the help on the parameter.